Update readme (#3)
DevExpress-Examples/xaf-how-to-access-a-tab-control-in-a-detail-view-layout: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E372)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# How to access a tab control in a Detail View layout

> This example demonstrates how to implement a View Controller that accesses layout controls and customize them as required. In particular, the second tab in the tab control should become active when the detail form is opened.

Author: ```suggestion
This example implements a View Controller that accesses layout controls and customizes them as required. In particular, the second tab in the tab control should become active when the detail form is opened.

```

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E372)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # How to access a tab control in a Detail View layout

Author: ```suggestion
# XAF - How to access a tab control in a Detail View layout

```

<!-- default badges end -->
# XAF - How to access a tab control in a Detail View layout

This example implements a View Controller that accesses layout controls and customizes them as required. In particular, the second tab in the tab control should become active when the detail form is opened.

**WinForms** 

<kbd><img width="601" alt="Du2Er0SPZV" src="https://user-images.githubusercontent.com/14300209/226934893-008269a8-66b7-4f3e-a04a-d1a000d86e83.png"><kbd>

**ASP.NET Core Blazor**

<kbd>![|image](https://user-images.githubusercontent.com/33227868/231761563-86aaa8fa-ebaf-4b20-958e-a4190fafcfa4.png)</kbd>

## Implementation Details

1. This functionality is implemented in the [WinCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Win/Controllers/WinCustomizeTabControlViewController.cs) and [BlazorCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Blazor.Server/Controllers/BlazorCustomizeTabControlViewController.cs) classes, added to the WinForms and Blazor application projects respectively. Copy these classes to the appropriate projects in your test solution.

> 2. To identify the layout element by the **TabbedGroup** string, make sure this identifier is specified under the **Views | YourBusinessObject_DetailView | Layout** node in the Model Editor (invoked for the _YourSolutionName.Module/Model.DesignedDiffs.xafml_ file).

Reviewer: by the **MyTabbedGroup** string

Author: ```suggestion
2. To identify the layout element by the **MyTabbedGroup** string, make sure this identifier is specified under the **Views | YourBusinessObject_DetailView | Layout** node in the Model Editor (invoked for the _YourSolutionName.Module/Model.DesignedDiffs.xafml_ file).

```

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E372)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# XAF - How to access a tab control in a Detail View layout

This example implements a View Controller that accesses layout controls and customizes them as required. In particular, the second tab in the tab control should become active when the detail form is opened.

**WinForms** 

<kbd><img width="601" alt="Du2Er0SPZV" src="https://user-images.githubusercontent.com/14300209/226934893-008269a8-66b7-4f3e-a04a-d1a000d86e83.png"><kbd>

**ASP.NET Core Blazor**

<kbd>![|image](https://user-images.githubusercontent.com/33227868/231761563-86aaa8fa-ebaf-4b20-958e-a4190fafcfa4.png)</kbd>

## Implementation Details

> 1. This functionality is implemented in the [WinCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Win/Controllers/WinCustomizeTabControlViewController.cs) and [BlazorCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Blazor.Server/Controllers/BlazorCustomizeTabControlViewController.cs) classes, added to the WinForms and Blazor application projects respectively. Copy these classes to the appropriate projects in your test solution.

Reviewer 2: ```suggestion
1. This functionality is implemented in the [WinCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Win/Controllers/WinCustomizeTabControlViewController.cs) and [BlazorCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Blazor.Server/Controllers/BlazorCustomizeTabControlViewController.cs) classes, added to the WinForms and Blazor application projects respectively. Copy these classes to the corresponding projects in your test solution.

```
